### DISCUSSION TITLE
Incoming Messages Not Received

### DISCUSSION THREAD

**Discussion by JSelway** [postalserver/postal#2645] [Help with using Postal] [answered]
Hi,

I've set up a Postal installation, with green lights for the organization's mail server domain - e.g somedomain.org.uk. I am able to send messages from Postal, however receiving messages are not working.

I can simulate an incoming message to a route in the UI successfully, as well as sending a message to the alternate Address in the route but all attempts to send a message to the Name in the route result in an undeliverable:

Remote Server returned '554 5.1.2 <mx.postal.somedomain.org.uk #5.1.2>'

Searching here and wider has not yielded any results I can use, so any help would be appreciated.

**Comment by willpower232** [comment]
I don't believe Postal outputs code 554, is your Postal definitely running? Do you definitely not have exim or postfix claiming the port before docker can?

**Reply by JSelway** [reply]
Don't believe so, Postal can receive mail using the Address value 'routes.postal.xxx' , just not directly to the mail address.

**Reply by willpower232** [reply]
is the DNS for mx.postal.somedomain.org.uk correct? you can double check with whatsmydns.net to make sure the whole world is seeing what you think it is

**Reply by JSelway** [reply]
Yeah, I think DNS is the issue. 
So, I have an MX record at somedomain.org.uk pointing to mx.postal.somedomian.org.uk but no records for that mx.postal... There is an A record for postal.somedomain.org.uk pointing to my postal server IP
Am I right in thinking I need an A record for mx.postal.somedomain.org.uk pointing to IP? As it can't reconcile mx.postal to an IP.

**Reply by willpower232** [reply, marked as answer]
Yeah you would need to do that, the instructions changed at some point to remove `mx.` as a configuration https://docs.postalserver.io/install/dns-configuration#example-postal-configuration

**Reply by JSelway** [reply]
Thanks for your help, it was the DNS. 
I removed the 'mx.' from the record, so it points to postal. and I can now see it working.
